docs/pt-br.md

```diff
@@ -0,0 +1,2460 @@
+---
+title: Brazilian Portuguese Version
```

Comment:
Sorry for being ignorant but is there a difference between Portugese and Brazilian Portugese?

Reply:
There are some differences between Portuguese from Portugal and Portuguese from Brazil. Here are some of them:

- Pronunciation: Brazilian Portuguese is more musical and clear, while Portuguese from Portugal is faster and the vowels are harder.
- Spelling: Despite an attempt at unification in 1990, there are still some spelling differences between the two varieties.
- Vocabulary: There are significant differences in vocabulary, which can sometimes cause misunderstandings.
- Grammar: The main grammatical difference is the use of the second person singular pronoun.

Despite the differences between Portuguese in Portugal and Brazil, communication between speakers of the two languages is generally clear. The differences are similar to those between British and American English. Although there may be some words or expressions that are specific to one region and not known in another, the context of the conversation often helps clarify the meaning.